PR for BXMSDOC-7792: Single-source and add "Artifact management" and "Archetype management" chapters in the upstream community doc. (#125)
kaldesai/kie-docs: doc-content/shared-kie-docs/src/main/asciidoc/Workbench/ArchetypeManagement/proc-business-central-additional-features-archetype.adoc

.Procedure

> . In {CENTRAL}, select the *Admin* icon in the upper-right corner of the screen and select *Archetypes*.

Reviewer: ```suggestion
. In {CENTRAL}, select the *Admin* icon and then select *Archetypes*.
```
[Style Guide - Location of interface elements](https://source.redhat.com/groups/public/ccs/ccs_folder/ibm_style_guide/ibm_style_062121_q2_2021pdf) - "When you refer to the location of an element in a graphical user interface, do not use sensory words, such as top, bottom, right, or left. Instead of sensory words, refer to the element by its label or tooltip"

Author: Noted
